### ISSUE TITLE
How to run this model for a custom dataset?

### ISSUE THREAD

**Issue by sainatarajan** [nv-tlabs/GSCNN#21]
Hi!, Thanks for the fantastic paper and also for providing code to it. Can you tell me how to run this model for my custom dataset? My training dataset has a set of images and it's masks and I would like it to predict the masks for the test dataset.

**Comment by tovacinni** [comment]
Thanks for your interest in our work!

If you want to run it on a custom dataset, you'll need to implement your own dataloader and such. Refer to some of the code in datasets/ for an example of how you should do this! 

If there's a specific dataset you have in mind, I may be able to assist at least a little bit in terms of how to integrate it.
